### DISCUSSION TITLE
logging out of shopify admin dashboard doesn't  log out the App

### DISCUSSION THREAD

**Discussion by aruntommy** [osiset/laravel-shopify#859] [Q&A]
when I run my App url after logging out of  shopify admin dashboard logged user session still exist in my app  this prevents me from installing app in another store because logging out of shopify does not affect the app. logged session still exist in app. how do i destroy my app session when the shopify is logged out and redirect to install page

**Comment by squatto** [comment]
They are completely separate authentications - your app has no way of knowing if the admin user is logged in to the Shopify admin. What you could do is force your app users to log out after a few minutes of inactivity, but that is immensely heavy-handed and just ends up punishing your regular users.

What is happening when you try to install the app in another store? The package creates a `User` with the email address `shop@{admin-domain}.myshopify.com`, so installing the app on another store _should_ authenticate a completely different user. Is that not happening?
